src/sentry/static/sentry/app/views/performance/constants.tsx

```diff
@@ -2,7 +2,7 @@ import {t} from 'app/locale';
 
 export const PERFORMANCE_TERMS: Record<string, string> = {
   apdex: t(
-    'Apdex is a ratio of satisfactory response times to unsatisfactory response times.'
+    'Apdex is the ratio of satisfactory or tolerable response times to all response times.'
```

Comment:
Alberto commented that `both... and` might be better grammar.

```suggestion
    'Apdex is the ratio of both satisfactory and tolerable response times to all response times.'
```

Reply:
This edit sounds good.